### PR TITLE
docs: add story about configuring HMR

### PR DESCRIPTION
**Pull request by beeman**
This story explains how to configure HMR for an Angular app. 

A link to this story could be used in the warning text that appears when running `ng serve --hmr`.

### CODE REVIEW

File: docs/documentation/stories/configure-hmr.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+```json
+"scripts": {
+  ...
+  "hmr": "ng serve --hmr -e=hmr"
+}
+```
+
+
+### Add dependency for @angular/hmr and configure app
```
Comment (intellix) [inline docs/documentation/stories/configure-hmr.md]: @angular/hmr doesn't exist and it confused me :)

**Comment by intellix** [comment]
Maybe change this link to point to this story instead? https://github.com/angular/angular-cli/blob/2be0bf2d533e75617da0db034ffc2dc9cd5555db/packages/%40angular/cli/tasks/serve.ts#L79

File: docs/documentation/stories/configure-hmr.md (hunk 1 above)
Comment (beeman) [inline docs/documentation/stories/configure-hmr.md]: Thanks @intellix , updated 👍

**Comment by beeman** [comment]
Hey @filipesilva, thanks for reviewing. 

I updated the index you mentioned and rebased off master.

**Comment by talamaska** [comment]
Hello, I'm kinda confused on what's the right way of setting up HMR. I read your guide and the one provide by angularclass/angular-hmr. Both are completely different. The one provided by you is really simple, but lacks of demonstration how it would work with some store in my case ngrx/store. The one from angularclass, uses some hooks in order to do same things plus coordinate the store, delete ng generated styles and etc. So I wander, is it possible to do the same with your example and how. probably this has nothing to do with angular-cli, but anyways I want to understand why there are those both ways of setting up HMR and which one is correct

**Pull request merged**